core/src/sync/state/snapshot_chunk_sync.rs

```diff
@@ -653,14 +658,15 @@ impl SnapshotChunkSync {
             .expect("trusted_blame_block header must exist");
 
         // check checkpoint position in `state_blame_vec`
-        let blame_vec_offset = trusted_blame_block.height() as usize
-            - checkpoint.height() as usize;
+        let blame_vec_offset = (trusted_blame_block.height()
```

Comment:
I remembered that I saved the offset in the Inner struct. If it's the same offset for receipt can we reuse it?

Reply:
BTW, in restore_execution_state I added DEFERRED_STATE_EPOCH_COUNT to inner.blame_vec_offset, which should be updated as well.

Reply:
true_state_root_by_blame_info corresponds to the offset without DEFERRED_STATE_EPOCH_COUNT.